lib/get-html.ts

```diff
@@ -29,7 +29,25 @@ export const getHtml = async (
 
   // get only the HTML we care about to avoid unnecessary cache updates
   $('[data-testid="callout"]').remove();
-  const html = $("#article-contents").parent().parent().html() ?? "";
+  const data = $("#article-contents").parent().parent();
+  data.find("*").each((i, el) => {
+    const classes = $(el).attr("class");
+
+    if (classes) {
+      const filteredClasses = classes
+        .split(" ")
+        .filter((classSelector) =>
+          ["language-json", "warning"].includes(classSelector)
+        );
+
+      if (filteredClasses.length) {
+        $(el).attr("class", filteredClasses.join(" "));
+      } else {
+        $(el).removeAttr("class");
+      }
+    }
+  });
```

Comment:
could you add a comment about why we are doing this, maybe with a reference to https://github.com/octokit/webhooks/issues/642?

Reply:
Added a comment 510feacc3be13e095d372a93e219827644a0d42a